<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" default="192.168.0.121"/>
  <arg name="load_gripper" default="false" />
  <rosparam param="/customized_gravity_direction">
   [7.5439190864562988,-6.2554783821105957,-0.4408954679965973]
   </rosparam>
   <rosparam param="/tool_vector">
   [-0.01, 0.0, 0.03]
   </rosparam>
   <param name="/tool_mass" value="0.73"/>

  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find franka_controllers)/config/franka_controllers.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="joint_impedance_controller"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_example_controllers)/launch/robot.rviz"/>
</launch>